feature-libs/asm/components/csagent-login-form/csagent-login-form.component.html

```diff
@@ -32,13 +32,8 @@
   </button>
 </form>
 
-<div
+<cx-dot-spinner
```

Comment:
I just realized that this HTML change needs documented as well.

Reply:
Done